runtime/src/accounts_hash.rs

```diff
@@ -1449,34 +1459,34 @@ pub mod tests {
             AccountsHash::compare_two_hash_entries(&val, &val5)
         );
     }
+    /*
+        fn test_de_dup_accounts_in_parallel(
+            account_maps: &[CalculateHashIntermediate],
+        ) -> (Vec<&Hash>, u64, usize) {
+            AccountsHash::default()
+                .de_dup_accounts_in_parallel(&vec![vec![account_maps.to_vec()]][..], 0)
+        }
+        #[test]
+        fn test_accountsdb_remove_zero_balance_accounts() {
```

Comment:
Should these tests be commented out? Are they broken?

Reply:
Doh. nope. changed signatures and was trying to see how fast it would run. I'll fix that.

Reply:
removed the /*... and fixed the test and function.